### PR TITLE
Update moabs to 1.3.8.2

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`moabs`](https://bioconda.github.io/recipes/moabs/README.html): **1.3.8.1** &rarr; **1.3.8.2**

[install with bioconda](http://bioconda.github.io/recipes/moabs/README.html) [Conda](https://anaconda.org/bioconda/moabs/files)

Info | Link
-----|-----
Recipe | [`recipes/moabs`](https://github.com//bioconda/bioconda-recipes/tree/bump/moabs/recipes/moabs) (click to view/edit other files)
Releases |[https://github.com/sunnyisgalaxy/moabs/tags](https://github.com/sunnyisgalaxy/moabs/tags)
Author | `@sunnyisgalaxy`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/moabs/meta.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ package:
   version: {{ version }}
 
 build:
+  skip: True  # [osx]
```
Comment (lijinbio) [inline recipes/moabs/meta.yaml]: Hi @dpryan79 , I realized you have added this `skip` option to build. May I ask what does this option intends to do? Thank you.

Comment (dpryan79) [reply recipes/moabs/meta.yaml]: It skips building on OSX, since it seems this doesn't deal with clang well.

Comment (lijinbio) [reply recipes/moabs/meta.yaml]: If it skips building on OSX, can this package be installed on OSX by conda?

**Comment by dpryan79** [comment]
@bioconda-bot please merge

File: recipes/moabs/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/moabs/meta.yaml]: No

**Pull request merged**